src/main/app/src/app/admin/mailtemplate/mailtemplate.component.ts

```diff
@@ -75,39 +76,42 @@ export class MailtemplateComponent extends AdminComponent implements OnInit, Aft
         .label(this.fields.NAAM)
         .validators(Validators.required)
         .build();
-        this.mailFormField = new SelectFormFieldBuilder(this.template.defaultMailtemplate ? mail.label : mail)
+        this.mailFormField = new SelectFormFieldBuilder(mail ? mail.label : mail)
         .id(this.fields.MAIL)
         .label(this.fields.MAIL)
-        .readonly(this.template.defaultMailtemplate)
+        .readonly(mail != null)
```

Comment:
Op regel 79 vergeleek je niet expliciet met null. Dan hoeft dat hier ook niet. Dus readonly(mail)

Reply:
Dat werkt helaas niet, in een ternary operator snapt hij dat het een waarde moet hebben, maar als losse waarde meegeven, dan doet hij die check niet.